java/lib/NetworkTablesJNI.cpp

```diff
@@ -773,6 +958,14 @@ JNIEXPORT jdoubleArray JNICALL Java_edu_wpi_first_wpilibj_networktables_NetworkT
 JNIEXPORT jobjectArray JNICALL Java_edu_wpi_first_wpilibj_networktables_NetworkTablesJNI_getStringArray__Ljava_lang_String_2_3Ljava_lang_String_2
   (JNIEnv *env, jclass, jstring key, jobjectArray defaultValue)
 {
+  if (!key) {
+    env->ThrowNew(nullPointerEx, "key cannot be null");
+    return nullptr;
+  }
+  if (!defaultValue) {
```

Comment:
I don't think we should throw on a null defaultValue. I know teams in the past have used null as their defaultValue since it's one of the cases that doesn't often get returned from ntcore, so its easy to check.